packages/edit-post/src/components/header/header-toolbar/index.js

```diff
@@ -77,7 +77,10 @@ function HeaderToolbar( { onToggleInserter, isInserterOpen } ) {
 			<EditorHistoryUndo />
 			<EditorHistoryRedo />
 			<TableOfContents hasOutlineItemsDisabled={ isTextModeEnabled } />
-			<BlockNavigationDropdown isDisabled={ isTextModeEnabled } />
+			<BlockNavigationDropdown
+				isDisabled={ isTextModeEnabled }
+				__experimentalWithEllipsisMenu
```

Comment:
I don't think this should be enabled here just yet while there are focus and keyboard navigation issues outstanding.